annotated_test.go

```diff
@@ -1347,6 +1347,78 @@ func TestAnnotate(t *testing.T) {
 		assert.Contains(t, err.Error(), "encountered error while applying annotation using fx.Annotate to go.uber.org/fx_test.TestAnnotate.func1(): cannot apply more than one line of ResultTags")
 	})
 
+	t.Run("invalid ParamTag quotations", func(t *testing.T) {
```

Comment:
+1 to @abhinav 's comment about additional error cases that this wouldn't cover.

I also think we can convert these tests to table tests since much of the logic is the same between tests (https://github.com/uber-go/guide/blob/master/style.md#test-tables)

i.e. the test case struct could look something like

```
type test struct {
    name string
    giveTags string
    giveAnnotation fx.Annotation // set this to either fx.ParamTags or fx.ResultTags
    wantErr string
}
```